kqueen_ui/asset/dynamic/scss/main.scss

```diff
@@ -77,8 +77,9 @@ a.disabled {
 }
 
 .cluster-status {
-  width: 50%;
+  width: 80%;
```

Comment:
Actually, I think it takes too much attention. With 50% it was more tidy

Reply:
Now it differs from buttons

Reply:
It differs from buttons in any case